### ISSUE TITLE
callback is not defined

### ISSUE THREAD

**Issue by vanodevium** [tronprotocol/tronweb#429]
Hi!

I have a function:
```js
const fee = process.env.FEE_LIMIT || 30000000;

async function payTRC20(toAddress, amount, fromAddress, tronWeb, contract) {
  const options = {
    feeLimit: fee,
    callValue: 0,
  };
  const tx = await tronWeb.transactionBuilder.triggerSmartContract(
    contract,
    "transfer(address,uint256)",
    options,
    [
      {
        type: "address",
        value: toAddress,
      },
      {
        type: "uint256",
        value: amount * 1000000,
      },
    ],
    tronWeb.address.toHex(fromAddress),
  );
  const signedTx = await tronWeb.trx.sign(tx.transaction);
  return tronWeb.trx.sendRawTransaction(signedTx);
}

module.exports = payTRC20;
```

This function works fine, but for some wallets it throws an error with the text:
```
callback is not defined
```

---

Version:
"tronweb": "5.3.0"

---

Please tell me what the problem might be and how to fix it.

> An important clarification: if you repeat on such an "incorrect" wallet, the error will be the same. If you transfer money to such a wallet through a mobile application, everything goes without errors.

**Comment by vanodevium** [comment]
As far as I could figure it out, problem is in two places: 
- [1337](https://github.com/tronprotocol/tronweb/blob/master/src/lib/transactionBuilder.js#L1337)
- [1360](https://github.com/tronprotocol/tronweb/blob/master/src/lib/transactionBuilder.js#L1360)

This variable **callback** is not defined really.

---

**But still, I'm really asking you to tell me why only on some wallets I see this error, even though the function code is the same for all.**

**Comment by start940315** [comment]
We have known this bug and it would not be a problem in v6. For now, I recommend you to console.log the toAddress and amount variable to check if there are some error data.

**Comment by vanodevium** [comment]
Thanx! But main question is: why some wallets have this error?

**Comment by start940315** [comment]
Maybe they are using different version. Previous versions don't have this problem.

**Comment by Ar1stotele** [comment]
Hello, have you resolved this issue?
If not - change this line of the code `value: amount * 1000000` to `value: amount,` (add multiplication somewhere else). For me, it worked.

Good luck.

**Comment by start940315** [comment]
Fixed in TronWeb v6.0.0-beta.0. Welcome to try.
